packages/alice/lib/utils/alice_parser.dart

```diff
@@ -79,8 +79,10 @@ class AliceBodyParser {
 
   /// Get content type from [headers]. It looks for json and if it can't find
   /// it, it will return unknown content type.
-  static String? getContentType(
-      {required BuildContext context, Map<String, dynamic>? headers}) {
+  static String? getContentType({
+    required BuildContext context,
+    Map<String, dynamic>? headers,
```

Comment:
The `headers` should probably be a `Map<String, String>` to align with https://pub.dev/documentation/http/latest/http/BaseRequest/headers.html

Reply:
Not sure about that.

For example Dio defines headers as:
```dart
  Map<String, dynamic> get headers => _headers;
```

Reply:
I think we should stick with <String,dynamic> for now.

Reply:
Have you ever seen a header that's not a `String`?

Reply:
> For example Dio defines headers as

Dio is non-standard.

Reply:
Nope. https://stackoverflow.com/questions/34152142/possible-types-of-a-http-header-value 

You're right, but still, I'm curious why Dio uses dynamic.

Reply:
This is interesting https://pub.dev/documentation/dio/latest/dio/Headers-class.html

Only `String`s

Reply:
https://pub.dev/documentation/dio/latest/dio/Options/headers.html Not only strings :)

Reply:
What's more that look how the headers are defined in Dio https://pub.dev/documentation/dio/latest/dio/RequestOptions/headers.html

and then look at that method

https://github.com/cfug/dio/blob/eb1684e41812cf1774031cec3c014b444b3e6fb4/dio/lib/src/utils.dart#L145

Reply:
Okay, looks like they don't use any standard for headers. Let's use <String,String> on our side.